src/shogun2-core/src/main/java/de/terrestris/shogun2/service/AbstractCrudService.java

```diff
@@ -102,4 +103,26 @@ public void delete(E e) {
 		dao.delete(e);
 	}
 
+	/**
+	 * Clones an entity by detaching it from the hibernate session and resetting the ID to null.
+	 * The clone will be persisted as a new entity if persist is true.
+	 *
+	 * @param e The entity to clone
+	 * @param persist whether or not the clone should be persisted as a new entity
+	 *
+	 * @throws IllegalAccessException
+	 * @throws NoSuchFieldException
+	 */
+	@PreAuthorize("hasRole(@configHolder.getSuperAdminRoleName()) or hasPermission(#e, 'READ')")
+	public E cloneEntity(E e, boolean persist) throws NoSuchFieldException, IllegalAccessException {
+		dao.evict(e);
+
+		IdHelper.setIdOnPersistentObject(e, null);
```

Comment:
Wasn't this method once only used internally for tests? Now we have apublic method calling it...